README.md

```diff
@@ -36,6 +36,8 @@ need to serve them from a proper http server for it to work. An easy way to do t
 In order to compile itself `doc-gen4` requires:
 - a Lean 4 or `elan` installation
 - a C compiler if on Linux or MacOS (on Windows it will use Lean's built-in clang compiler)
+- for `references.bib` parsing, an installation of `pybtex` (https://pybtex.org/) is needed,
```

Comment:
This should a) give clear instruction on what to install and b) the instructions should force the user to install one particular version.

Reply:
> a) give clear instruction on what to install

Install the package `pybtex` if you're using Linux and the package manager have this; or install python/pip and run `pip install pybtex`. Is this OK?

> b) the instructions should force the user to install one particular version.

I'm not sure about this, perhaps a version which is not too old should work, since we are only calling command line tools, not calling libraries.

Reply:
If we go through with the python approach (which I'm still not seeing) I would prefer to have a clear:
```
pip install pybtex==x.y.z
```
with a clear version that we have tested and can guarantee works. Doing just `pip install` or using a package manager has the potential of causing behavior on other people's machines that we do not know about due to version differences.

Reply:
I've updated the README, now `pybtex` is optional, as the built-in bibtex parser is work in progress. I hope this is OK.